vocabularies/trs.ttl

```diff
@@ -0,0 +1,178 @@
+  skos:prefLabel "Hebrew Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:IndianNationalCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <https://aa.usno.navy.mil/faq/docs/calendars.php> ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Indian_national_calendar> ;
+  skos:definition "The Indian national calendar, sometimes called the Shalivahana Shaka calendar is used, alongside the Gregorian calendar, by The Gazette of India, in news broadcasts by All India Radio and in calendars and communications issued by the Government of India." ;
+  skos:prefLabel "Indian National Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+<http://www.opengis.net/def/trs/BIPM>
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <http://www.opengis.net/def/trs/BIPM> ;
+  rdfs:seeAlso <https://www.bipm.org/en/bipm/tai/tai> ;
+  skos:altLabel "BIMP" ;
+  skos:altLabel "TAI" ;
+  skos:definition "International Atomic Time (TAI), is a time scale calculated at the Bureau International des Poides et Mesures (BIPM) using data from some two hundred atomic clocks in over fifty national laboratories. TAI is a uniform and stable time scale which does not, therefore, keep in step with the slightly irregular rotation of the Earth. For public and practical purposes it is necessary to have a scale that, in the long term, does. Such a scale is Coordinated Universal Time (UTC), which is identical with TAI except that from time to time a leap second is added to ensure that, when averaged over a year, the Sun crosses the Greenwich meridian at noon UTC to within 0.9 s. The dates of application of the leap second are decided by the International Earth Rotation Service (IERS)." ;
+  skos:prefLabel "International Atomic Time" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:IranianCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Iranian_calendars> ;
+  skos:definition """The Iranian calendars or Iranian chronology (Persian: گاه‌شماری ایرانی‎, Gāh-Shomāri-ye Irāni) are a succession of calendars invented or used for over two millennia in Iran also known as Persia. One of the longest chronological records in human history, the Iranian calendar has been modified time and time again during its history to suit administrative, climatic, and religious purposes.
+
+As one of the few calendars designed in the era of accurate positional astronomy, the Iranian calendar uses a very complex leap year structure which makes it the most accurate solar calendar in use today.""" ;
```

Comment:
Is this still part of the definition? Should be on a single line.
Also ends with many quotation marks.

Reply:
I believe the triple quote marks at the start and end of definitions allow for non-standard characters and quote a marks to exist within the definition without creating errors.

Reply:
Thanks for clearing that up Vance!

Reply:
Not really for "non-standard characters", more line breaks, see the formal definition: [turtle-literals](https://www.w3.org/TR/turtle/#turtle-literals)